test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -349,9 +360,14 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 				By("Verifying node kernel arguments")
 				cmdline, err := nodes.ExecCommand(context.TODO(), &workerRTNodes[0], []string{"cat", "/proc/cmdline"})
 				Expect(err).ToNot(HaveOccurred())
-				Expect(cmdline).To(ContainSubstring("intel_pstate=passive"))
-				Expect(cmdline).ToNot(ContainSubstring("intel_pstate=active"))
 
+				if isIntel {
+					Expect(cmdline).To(ContainSubstring("intel_pstate=passive"))
+					Expect(cmdline).ToNot(ContainSubstring("intel_pstate=active"))
+				} else {
```

Comment:
Would isAMD be more descriptive with an else that throws an error? Just in case we get ARM machines in the future?

Reply:
Ack, i will add it

Reply:
fixed in the latest commit